landavailability/api/management/commands/utils.py

```diff
@@ -0,0 +1,12 @@
+from api.models import Address, CodePoint
+
+
+def get_address_from_postcode(postcode):
+    return Address.objects.filter(postcode=postcode)
```

Comment:
Does the input need normalising here somewhere?  Like .replace(' ', '').upper() or something?